newIDE/app/src/AssetStore/index.js

```diff
@@ -80,6 +90,37 @@ export const AssetStore = ({ project }: Props) => {
   ] = React.useState<?PrivateAssetPackListingData>(null);
   const { onPurchaseSuccessful } = React.useContext(AuthenticatedUserContext);
 
+  // The saved scroll position must not be reset by a scroll event until it
+  // has been applied.
+  const hasAppliedSavedScrollPosition = React.useRef<boolean>(false);
+  // The saved scroll position must not be applied after users have scrolled.
+  const hasScrolled = React.useRef<boolean>(false);
+  const needScrollUpdate = React.useCallback(() => {
```

Comment:
I feel like this variable name is a check to know if something needs a scroll update although it defines there's a need. What do you think about `setScrollUpdateNeeded` instead?